src/components/VueHotspot.vue

```diff
@@ -34,24 +34,21 @@
 </template>
 
 <script>
-import Vue from 'vue'
 import DataPoint from './module/DataPoint.vue'
 import ControlBox from './module/ControlBox.vue'
 import { throttle } from './utils/common.js'
-import VueCompositionApi, {
-  createComponent,
+import {
+  defineComponent,
   ref,
   reactive,
   toRefs,
   isRef,
   onMounted,
   onUnmounted,
   watch
-} from '@vue/composition-api'
+} from 'vue-demi'
 
-Vue.use(VueCompositionApi)
```

Comment:
According to https://github.com/vueuse/vue-demi#install, should we `install()` here?